sdk-java/src/main/java/com/spotify/confidence/Confidence.java

```diff
@@ -173,21 +183,21 @@ public <T> FlagEvaluation<T> getEvaluation(String key, T defaultValue) {
     }
   }
 
-  CompletableFuture<ResolveFlagsResponse> resolveFlags(String flagName) {
+  public CompletableFuture<ResolveFlagsResponse> resolveFlags(String flagName) {
     return client().resolveFlags(flagName, getContext());
   }
 
   @VisibleForTesting
-  static Confidence create(
+  public static Confidence create(
```

Comment:
This method should not be exposed, but it's marked as VisibleForTesting at least